documentation/specs/enable-binlog-collection-by-env-var

```diff
@@ -0,0 +1,113 @@
+## Purpose
+
+Enable binary logging in CI/CD pipelines when `-noAutoResponse` blocks response files, without modifying build scripts or project files.
```

Comment:
Think that this is a problem even without `-noAutoResponse`. Using an RSP file would necessitate writing artifacts to disk or modifying an existing RSP file in place. The goal I was thinking of is how can you enable collection without touching the artifacts on disk at all.